ci/kind/kind-setup.sh

```diff
@@ -300,6 +300,19 @@ function configure_vlan_subnets {
       docker_run_with_host_net iptables -t filter -A FORWARD -i ${vlan_interfaces[j]} -o ${vlan_interfaces[i]} -j ACCEPT
     done
   done
+
+  # With FlexibleIPAM, Antrea SNAT is disabled (noSNAT: true) so Pods don't have access to the external network by default (including regular / NodeIPAM Pods). 
+  # With our configuration, Docker will already perform SNAT for FlexibleIPAM Pods. However, we need a custom SNAT rule for regular Pods.
```

Comment:
Reading this comment that i suggested again, I think this part may be inaccurate:

> With our configuration, Docker will already perform SNAT for FlexibleIPAM Pods

I am not sure what I was thinking about when I suggested this. I cannot see why that would be the case now. What do you think (could be checked with `iptables-save -t nat`)? A more accurate comment for the second iptables rule would be:

> With FlexibleIPAM, Antrea SNAT is disabled (noSNAT: true) so Pods don't have access to the external network by default (including regular / NodeIPAM Pods).
> Our e2e tests require external network access for regular Pods, so we need to add a custom SNAT rule.

I just changed the second line. Is that accurate ^ ?

Reply:
I think yes, 2nd SNAT rule we are adding to access external network for regular pod based on e2e tests. I updated the comment, thanks!